.gitignore

```diff
@@ -30,3 +30,4 @@ docs/
 
 # Debug Logs
 debug.log
+features/polyTalkPreview/dist
```

Comment:
You should probably just dismiss this locally. It's annoying, I know, but this .gitignore has a higher calling.

Reply:
Yeah, I can do that.

Reply:
I will just keep dist in. It won't hurt anyone.

Reply:
> I will just keep dist in. It won't hurt anyone.

I'm not sure dist does not have genuine content somewhere; it could have. IIRC, we've not done it in the past for that reason.